playbook.md

```diff
@@ -971,189 +971,72 @@ to build an appropriate hosting platform for their service. This is fair
 enough, because they shouldn't have to. We built GovPress to meet this need,
 and it is now used by most of our clients.
 
+We build [websites](https://www.digitalmarketplace.service.gov.uk/g-cloud/services/930612236449495), [campaign](https://www.digitalmarketplace.service.gov.uk/g-cloud/services/691308552308120) and [blogging](https://www.digitalmarketplace.service.gov.uk/g-cloud/services/355674790119695) platforms hosted on GovPress, configured to our client’s branding requirements.
+
 You can read more about GovPress at
 [www.govpress.com](https://www.govpress.com).
 
-We're in the process of building a new container-based hosting platform which
-will be able to support services that aren't built in WordPress. Watch this
-space.
+### Tailored support
+
+We also offer support for services that aren't built in WordPress. We are able to host and support services with our [tailored support plan for cloud applications](https://www.digitalmarketplace.service.gov.uk/g-cloud/services/393702576050692), even if your service isn't hosted with us.
```

Comment:
Wonder if there are actually two sections here - one for Dalmatian https://www.digitalmarketplace.service.gov.uk/g-cloud/services/754953588860938 
and one for the tailored support, as per your commit? 
Think it might make it a little clearer?

Reply:
Agreed. I've only added the tailored support plan and think there's another space here for Dalmation, separate from tailored support and govpress. 

I was unclear if Dalmatian was the final name of the product, so I have left this out for now - it feels like there should be more discussion before this gets added to the playbook.